src/gtest/test_joinsplit.cpp

```diff
@@ -22,6 +22,41 @@ using namespace libzcash;
 
 extern ZCJoinSplit* params;
 
+typedef std::array<JSDescription, 2> SproutProofs;
+// Make both the PHGR and Groth proof for a Sprout statement,
+// and store the results in JSDescription objects.
+SproutProofs makeSproutProofs(
+        ZCJoinSplit& js,
+        const std::array<JSInput, 2>& inputs,
+        const std::array<JSOutput, 2>& outputs,
+        const uint256& joinSplitPubKey,
+        uint256& out_randomSeed,
```

Comment:
This doesn't appear to be used, so it should be removed.

Reply:
Thanks! removed.